nextflow.config

```diff
@@ -249,6 +249,15 @@ params {
     bgc_hmmsearch_savetargets                       = false
     bgc_hmmsearch_savedomains                       = false
 
+    // RUNDBCAN options
+    run_cazyme_screening                                    = false
+
+    cazyme_skip_dbcan                                       = false
```

Comment:
Missing the parameter to support pre-downloaded/locally databases (as you've already given instructions for!)